lib/sword/mets/eprints_dc_mets_file.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Sword::Mets::EprintsDcMetsFile < Sword::Mets::MetsFile
+  include Sword::Mets::EprintsDcConstants
+
+  attr_accessor :abstract,
+                :bibliographic_citation,
+                :creators,
+                :date_available,
+                :identifier_uri,
+                :publisher,
+                :title,
+                :xpath_info
+
+  def initialize(nokogiri_xml_doc)
+    super
+    @xpath_info = XPATH_INFO
```

Comment:
(This is an optional suggestion)

I tend to avoid using `include` for constants because it makes it harder to find them when searching in a file, especially when you have multiple `include`s.  I like to just refer to the constant in its module, i.e. `Sword::Mets::EprintsDcConstants:: XPATH_INFO`.

Reply:
I did waffle between the option you mentions and how I finally coded it. I can ticket this as an investigation and possible future change.

Reply:
Ticket sounds good!  Just wanted to put the idea out there, but it's not urgent/important.